src/utils/QCCalculator.cpp

```diff
@@ -396,29 +391,48 @@ class TOPPQCCalculator :
     at.colTypes.push_back("MS:1000285");
     UInt max = 0;
     Size below_10k = 0;
-    for (Size i = 0; i < exp.size(); ++i)
-    {
-      if (exp[i].getMSLevel() == 1)
+    std::vector<OpenMS::Chromatogram> chroms = exp.getChromatograms();
+    if (chroms.size() == 1) //real TIC from the mzML
```

Comment:
-- General information --

File name: KT_Pan_171114_CWC2_XL_HF_picked_ID.mzML
File type: mzML

Peak type (metadata): Peak data
Peak type (estimated): Peak data

Number of spectra: 22403
Number of peaks: 9221142

Ranges:
  retention time: 0.10 .. 3600.06
  mass-to-charge: 110.00 .. 5990.14
  intensity:      509.27 .. 3882716672.00

MS levels: 1, 2
Number of spectra per MS level:
  level 1: 5931
  level 2: 16472

Number of chromatograms: 1
Number of chromatographic peaks: 2589

Number of chromatograms per type: 
  total ion current chromatogram:                         1

This information seems to be present... see FileInfo above